compiler/testData/debug/stepping/assertion.kt

```diff
@@ -0,0 +1,45 @@
+// FILE: test.kt
+public val MASSERTIONS_ENABLED: Boolean = true
+
+public inline fun massert(value: Boolean, lazyMessage: () -> String) {
+    if (MASSERTIONS_ENABLED) {
+        if (!value) {
+            val message = lazyMessage()
+            throw AssertionError(message)
+        }
+    }
+}
+
+
+public inline fun massert(value: Boolean, message: Any = "Assertion failed") {
+    if (MASSERTIONS_ENABLED) {
+        if (!value) {
+            throw AssertionError(message)
+        }
+    }
+}
+
+fun box(): String {
+    massert(true)
+    massert(true) {
+        "test"
+    }
+
+    return "OK"
+}
+
+// LINENUMBERS
```

Comment:
Test data of this test is affected by just merged #3157

Reply:
I was about to say this! :)

Reply:
Huh, you are absolutely right, though I don't think it's an improvement :) Steps are as they should be, will update.

Reply:
Well, for (inline) tests where multiple modules/files are involved, source names help you understand to which each line belongs.